install/centos-installation-guide.md

```diff
@@ -0,0 +1,52 @@
+# Installing Kata Containers on CentOS
+
+Note: Kata Containers is available for CentOS version 7.
+
+Note:
```

Comment:
Could you remove this 2nd "Note:". I think to be consistent with Ubuntu+Fedora the note need to show as:

> Note:
> Kata Containers is available for CentOS version 7.